xcell/cls/theory.py

```diff
@@ -2,21 +2,54 @@
 import numpy as np
 
 
+class ConcentrationDuffy08M500c(ccl.halos.Concentration):
+    """ Concentration-mass relation by Duffy et al. 2008
+    (arXiv:0804.2486) extended to Delta = 500-critical.
+    Args:
+        mdef (:class:`~pyccl.halos.massdef.MassDef`): a mass
+            definition object that fixes
+            the mass definition used by this c(M)
+            parametrization.
+    """
+    name = 'Duffy08M500c'
+
+    def __init__(self, mdef=None):
+        super(ConcentrationDuffy08M500c, self).__init__(mdef)
+
+    def _default_mdef(self):
+        self.mdef = ccl.halos.MassDef(500, 'critical')
+
+    def _check_mdef(self, mdef):
+        if (mdef.Delta != 500) or (mdef.rho_type != 'critical'):
+            return True
+        return False
+
+    def _setup(self):
+        self.A = 3.67
+        self.B = -0.0903
+        self.C = -0.51
+
+    def _concentration(self, cosmo, M, a):
+        M_pivot_inv = cosmo.cosmo.params.h * 5E-13
```

Comment:
Although easy to find, might you say that this is Eq. 4 in (arXiv:0804.2486)?

Reply:
Sure, done (but note that the paper was cited in the class docstring)